Notice.txt

```diff
@@ -10,29 +10,6 @@ purpose, commercial or non-commercial, and by any means.
 
 ----------------
 
-** Parsing PEM files from Bouncy Castle
```

Comment:
You need add a new entry for BouncyCastle NuGet package. Like we recently did for the new System.* NuGet packages we added for V4. https://github.com/aws/aws-sdk-net/blob/v4-development/Notice.txt#L127